### PR TITLE
[couchbase] implemented scan for couchbase

### PR DESCRIPTION
**Pull request by kruthar**
Adapted from #172

### CODE REVIEW

File: couchbase/src/main/java/com/yahoo/ycsb/db/CouchbaseClient.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2013 Yahoo! Inc. All rights reserved.
+ * Copyright (c) 2013 - 2016 Yahoo! Inc. All rights reserved.
```
Comment (busbey) [inline couchbase/src/main/java/com/yahoo/ycsb/db/CouchbaseClient.java]: 2013 Yahoo! Inc, 201? - 2016 YCSB Contributors

**Comment by busbey** [comment]
how do you want to handle attribution of authorship given the basis on #172?

**Comment by kruthar** [comment]
I'm not sure, I guess what I could do is start with the commit that this came from and then add a commit of my own to get things in the final state. That may introduce a bunch of irrelevant changes for one commit, but that way the original author has a commit in history. Would that work?

**Comment by busbey** [comment]
Sure, that would work. We could also do what some ASF projects do and have the author for the commit set to the original author and then list you in the commit message with

```
Amending-Author: First Last <email@example.com>
```

I'm fine with either approach.

**Comment by kruthar** [comment]
OK - license fixed, and cherry picked relevant changes from the original PR as the basis for this one.

**Comment by busbey** [comment]
+1

**Pull request merged**